pep-0526.txt

```diff
@@ -247,9 +262,14 @@ It's illegal to attempt to annotate ``global`` and ``nonlocal``::
   def f():
       global x: int  # SyntaxError
 
+  def g():
+      x: int  # Also a SyntaxError
+      global x
+
 The reason is that ``global`` and ``nonlocal`` don't own variables;
 therefore, the type annotations belong in the scope owning the variable.
 
+Only single assignment target and a single right hand side value is allowed.
```

Comment:
sorry to nitpick, but I think `Only single assignment targets and single right hand side values are allowed.` reads a little bit better grammar-wise.